operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -94,7 +95,8 @@ groups:
       severity: warning
   - alert: MimirIngesterRestarts
     annotations:
-      message: '{{ $labels.job }}/{{ $labels.instance }} has restarted {{ printf "%.2f" $value }} times in the last 30 mins.'
+      message: '{{ $labels.job }}/{{ $labels.instance }} has restarted {{ printf "%.2f"
+        $value }} times in the last 30 mins.'
```

Comment:
This line break in the middle looks very strange. Is that what linter and we want?

Reply:
See: https://github.com/grafana/mimir/pull/1606#issuecomment-1085739171